setup.py

```diff
@@ -21,10 +21,10 @@
 
 if not path.isfile(path.join(fortran_path, 'sami2py.x')):
     try:  # py27 does not have shutil.which()
-        cmd = ['gfortran', '-fno-range-check', '-fno-automatic', '-ffixed-line-length-none',
-               '-o', 'sami2py.x']
-        src = ['nrlmsise00_modified.f', 'grid-1.00.f', 'sami2py-1.00.f', 'hwm93.f', 'hwm07e_modified.f90',
-               'apexcord.f90', 'hwm14.f90']
+        cmd = ['gfortran', '-fno-range-check', '-fno-automatic',
```

Comment:
This I think is fine for now. A more general approach for the future to consider is to have a `sami2py.build()` function that is automatically run on `import sami2py` if the Fortran module isn't built yet. I use that technique on my packages. This allows me to not depend on `setuptools()` and to switch compilers and options seamlessly/automatically. 

But I think for now, probably most people have Gfortran and setuptools--just noting there's a more general way forward if it becomes an issue.